app/src/Components/Canvas/RectangleView.js

```diff
@@ -0,0 +1,93 @@
+export default class RectangleView {
+    constructor(x, y, w, h) {
```

Comment:
Maybe it could be better to store `x`, `y`, `w` and `h`?

Reply:
Fixed